main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,260 @@
+# ERTP Guide
+
+[ERTP](https://agoric.com/documentation/glossary/#ertp) (*Electronic
+Rights Transfer Protocol*)
+is Agoric's token standard for transferring tokens and other digital assets in
+JavaScript. Using the
+[ERTP API](https://agoric.com/documentation/ertp/api/),
+you can easily create and use digital assets, all of which are
+transferred exactly the same way and with exactly the same security properties. 
+
+ERTP uses
+*[object capabilities](https://agoric.com/documentation/glossary/#object-capabilities)*
+to enforce access control. If your program has a reference to an
+object, it can call methods on that object. If it doesn't have a
+reference, it can't. For more on object capabilities, see
+[Chip Morningstar's post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
+
+## Objects Overview
+
+![ERTP object relationships](./assets/relationships.svg)  
+
+There are eight fundamental ERTP objects, two of which are parts of
+another, and one which is used by that same object. For each entry,
+the name is linked to its primary page in this Guide. 
+
+- **[Amount](./Amounts.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#amount)**:
+  The description of an asset, it is not an asset itself, having no economic scarcity or economic value.
+  Made up of a:
+  - *[value](./Amounts.md)* *[(Glossary)](https://agoric.com/documentation/glossary/#value)*:  The
+  size of the amount.
+  - *[brand](./Amounts.md)* *[(Glossary)](https://agoric.com/documentation/glossary/#brand)* The
+  kind of asset. 
+  
+  You can think of these as the "how many" and the "what" of an asset, for example, 7 quatloos (an imaginary
+  currency).
+- **[AmountMath](./AmountMath.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#amountmath)**:
+Methods to do math operations on an `amount`..
+- **[Purse](./PursesAndPayments.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#purse)**: an
+  object for long term digital assets storage. 
+- **[Payment](./PuresAndPayments.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#payment)**:
+  an object for transfering digital assets to another party.
+- **[Mint](./IssuersAndMints.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#mint)**: an
+  object that creates new `payment` objects of a specific `brand`. Each `brand` has
+  a one to one relationship with a `mint` and vice versa.
+- **[Issuer](./IssuersAndMints.md)** **[(Glossary)](https://agoric.com/documentation/glossary/#issuer)**: 
+an object linked to a single `mint` object that creates empty `purse` objects
+for that mint's `brand` and operates on any `payment` of that `brand`. 
+
+## Fungible and Non-Fungible Assets
+
+There are two kinds of assets,
+*[fungible](https://agoric.com/documentation/glossary/#fungible)* and
```

Comment:
I think I'm going to leave this one. Working backwards, "non-fungible" gives an easy to use understandable term to use for them. I used "kind" for the nat, set, strSet to avoid overloading "type" and confusing it with JS typing and the like. I think a quick use of it in plain English  here, particularly since we never use "fungible" in ERTP itself, is OK.